src/app/find-help/accommodation-search/accommodation-search.component.html

```diff
@@ -1,5 +1,5 @@
 <mat-card>
-  <app-accommodation-search-form (search)="search($event)"></app-accommodation-search-form>
+  <app-accommodation-search-form (search)="search($event, { page: 0, size: 5 })"></app-accommodation-search-form>
```

Comment:
ja bym  to widział gdzieś w konfiguracji, zmieniamy wtedy w jednym miejscu,
np paginator-init.ts ?

Reply:
Przydałoby się w ogóle odkładać gdzieś wybraną paginację; to wkurza, jeśli użytkownik musi ją zmieniać przy każdym użyciu. Jeśli nie mamy do tego (i nie planujemy wkrótce) backendu, można by choćby użyć `localStorage`.

Reply:
ilość rekordów na stronę można dodać w localStorage, tutaj dużo roboty nie będzie

Reply:
Czy nie lipeiej to zeobic przez route params ?

Reply:
Jak cos to juz koncze tą funkcjonalnosc przez route params.

Reply:
Tem PR można zamknąc jak coś, bo robie to na innym branczu